packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/SelectWallet.tsx

```diff
@@ -46,6 +47,61 @@ export const SelectWallet = () => {
     });
   };
 
+  const getWalletCard = (wallet: IWalletForSelection) => {
+    const isDeleted = Boolean(wallet.isDeleted);
+    const isNotSecure = !wallet.hasPin;
```

Comment:
```suggestion
    const doesNotHavePin = !wallet.hasPin;
```
Try to not introduce new terms when not required